packages/phone-number-privacy/common/src/utils/logger.ts

```diff
@@ -1,12 +1,27 @@
-// tslint:disable: no-console
+import Logger, { createLogger, levelFromName, LogLevelString, stdSerializers } from 'bunyan'
+import bunyanDebugStream from 'bunyan-debug-stream'
+import { createStream } from 'bunyan-gke-stackdriver'
```

Comment:
Did you see who created this package?
https://www.npmjs.com/package/bunyan-gke-stackdriver
Thanks @mcortesi 😄

Reply:
😂 🙏 a true hero